tasks/Ultimatum game (various versions).md

```diff
@@ -0,0 +1,41 @@
+# Summary
+>  One player, the proposer, is endowed with a sum of money and can split it with another player (the responder). The responder then can accept or reject the money. 
+
```

Comment:
I would add a sentence or two about the consequences like "If the offer is accepted, both players split the money. If the offer is rejected, both players leave with nothing." also maybe clarify that it is usually played over several rounds.